src/bibx/_entities/collection_builders/cross_ref.py

```diff
@@ -12,4 +14,56 @@ def with_count(self, count: int):
         return self
 
     def build(self) -> Collection:
-        return Collection([])
+        url = f"https://api.crossref.org/works?query={self._query.lower().replace(' ', '+')}&filter=has-orcid:true,type:journal-article,has-references:true,from-pub-date:2003-01-01&rows={self._count}"
```

Comment:
I suggest you create a client and `type-check` the API response using pydantic, the client would look like this:

https://gist.github.com/odarbelaeze/8c9fa5b8735463c94137116af48a73e2